Tensile/KernelWriterAssembly.py

```diff
@@ -10963,11 +10965,11 @@ def emitAddressSetupCode(self, kernel, ss, tmpVgpr, tmpS01, edge, beta, atomic,
 
       # calculate global load offset
       # this has to be done after calculating new coord1
-      if not kernel["BufferStore"]:
+      if not kernel["BufferStore"] and (tc == 'D' or (tc == 'C' and beta)):
```

Comment:
It looks like GLOBAL_OFFSET C or D code is not generated for ldcEqualLdd and not beta case.
We need to change this part like either change this part of condition or call this function with D.

Reply:
Updated